src/pages/[platform]/how-amplify-works/index.mdx

```diff
@@ -159,13 +159,17 @@ Amplify makes it easy to quickly build web and mobile app user interfaces using
 
 With the Figma-to-code feature, you can convert Figma designs directly into React components. You can then connect these components to your cloud data in Amplify Studio, and customize them as needed in your code base. This automates the process of coding UIs from design mocks.
```

Comment:
@hibler13 I would limit this whole section to only JS and React vs just the links. Why tell users about Form builder and Figma to code if they can't use it?